### DISCUSSION TITLE
TailwindCSS PlayCDN not working

### DISCUSSION THREAD

**Discussion by 17Swagat** [tailwindlabs/tailwindcss#11449] [Help]
**What version of Tailwind CSS are you using?**
v3.3.2

**What build tool (or framework if it abstracts the build tool) are you using?**
None. Only plain HTML.

**What browser are you using?**
Edge, chrome, brave

**What operating system are you using?**
Windows

**Describe your issue**

I followed the official tailwindCSS docs and put the following code 
`<script src="https://cdn.tailwindcss.com"></script`
in my `index.html` file and nothing happened. Its just a simple code

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Playgron</title>
    <script src="https://cdn.tailwindcss.com"></script>
  </head>
  <body>
    <h1 class="text-red-100">
      Title
    </h1>
  </body>
</html>
```

**Comment by thecrypticace** [comment]
Hey, this is working for me in Chrome, Arc, Safari, FF, Edge, etc…

I noticed you chose a very bright color (`text-red-100`). It might appear to not be working because it's quite difficult to see against a white background at the default text size. Try testing with `text-red-600`.

**Reply by 17Swagat** [reply]
Yeah you are right, its working now :)
Sorry it was my stupid mistake :(
